.circleci/config.yml

```
@@ -3,10 +3,10 @@ version: 2.1
 parameters:
   slack-mentions:
     type: string
-    default: 'afiune,scottford'
+    default: "afiune,scottford"
```

Our new standard is to use the @allies Slack mention. 

JIRA: ALLY-267 
```suggestion
    default: "S01JP5A3ACQ"
```